app/src/main/java/ch/epfl/sdp/game/Client.java

```diff
@@ -195,4 +198,23 @@ private void sendUsedItems() {
             playerManager.getCurrentUser().getInventory().clearUsedItems();
         }
     }
+
+    /**
+     * Creates shelter area and coins on client side
+     * This is meant not to be synched between different devices (i.e. users in the same lobby won't see the same coin locations and shelter are locations
+     */
+    private void initCoinsAndShelterPoints() {
```

Comment:
So to be sure, the client and the server does not have the same coins and shelter area right? If it's correct, then maybe the shelter area can only have a Player instead of a list of Players, because it would never protect more than one player (the only thing we need to be sure is that the isShielded boolean is correctly fetched to the Server though). I'm sure we talked about this, but there is so much informations that I lose some in the way.